torch/distributions/uniform.py

```diff
@@ -64,7 +64,11 @@ def support(self):
 
     def rsample(self, sample_shape=torch.Size()):
         shape = self._extended_shape(sample_shape)
-        rand = self.low.new(shape).uniform_()
+        if torch._C._get_tracing_state():
+            # [JIT WORKAROUND] lack of support for .uniform_()
+            rand = torch.rand(shape, dtype=self.low.dtype, device=self.low.device)
+        else:
+            rand = self.low.new(shape).uniform_()
```

Comment:
Well this one can be just replace with the new-style constructor without any problems.

Reply:
OK, I'll combine these. (I wasn't sure if `uniform` was preferred over `rand`)